### ISSUE TITLE
Authoring Tool Interface Translation

### ISSUE THREAD

**Issue by mauroka** [adaptlearning/adapt_authoring#768]
Hello, I'd like to start translating the Authoring Tool Interface to Spanish.
According to the replay from Brian in this thread:
https://community.adaptlearning.org/mod/forum/discuss.php?d=828

The language pack for English is located here:
https://github.com/adaptlearning/adapt_authoring/blob/develop/routes/lang/en-application.json
In theory all we need is an es-application.json version for this in Spanish and to change the code in a few places.  Probably here and in the Gruntfile:
https://github.com/adaptlearning/adapt_authoring/blob/develop/frontend/src/core/app/app.js#L71

I've a doubt about the special chars like á or ñ. Should I use the html special chars codes in the translation of en-application.json? For exmple: configuration should be translated like configuraci&oacute;n?

Thanks.

**Comment by brian-learningpool** [comment]
Hi Mauro,

Add the special characters exactly as they are.  We can add a charset of UTF-8 to the response header to ensure these render as expected.

**Comment by brian-learningpool** [comment]
@mauroka, I was wondering did you get anywhere with this?

**Comment by mauroka** [comment]
Hi Brian.
I have the translated file. I didn't know the meaning of Tenant in the Authoring Tool so I left it like this. It just says tenant.
I can't attach the file here, so how would you want me to send it to you?

**Comment by brian-learningpool** [comment]
Hi Mauro,

That's excellent.  @dennis-learningpool has put together a great document on how to contribute here:
https://github.com/adaptlearning/adapt_authoring/blob/develop/CONTRIBUTING.md

I'm not sure how familiar you are with Git, but the main steps are:
* fork the project
* create a branch for the issue
* make your changes 
* push your changes 
* create a pull request via the GitHub website

**Comment by jpablo128** [comment]
Hello @mauroka did you get the translated file into the repository somewhere? I can help translate/review/etc. if you want.

**Comment by mauroka** [comment]
Hello @jpablo128 . The translation is finished, but I couldn't upload it to the repository. Would you like me to send it to your personal email so you could upload it?

**Comment by jpablo128** [comment]
Yes @mauroka , you can contact me through http://www.jpablo128.com/contact/ and I'll get back to you.

**Comment by taylortom** [comment]
Hi @jpablo128, @mauroka,
How did you get on with this?

**Comment by mauroka** [comment]
Hi @taylortom , I 've already pass the translated file to @jpablo128. He was going to upload it to GitHub.

**Comment by jpablo128** [comment]
Hi @taylortom, I did get the file from  @mauroka  and completed the missing word, but got busy with other things and didn't get to do the PR. I haven't been following the authoring tool side of things much... but  I'll try to review this again and do the PR tomorrow or during the weekend.

**Comment by taylortom** [comment]
Great :)

If it helps, I can add you as a collaborator to my fork of adapt_authoring, and you can push to that, or I can submit a PR for you?

**Comment by jpablo128** [comment]
Hi @taylortom , I sent a PR to your fork of adapt_authoring. I just added the file with the Spanish translation that @mauroka made initially (I've tweaked/completed some words). 
I've followed the guidelines for contributing, but I'm not sure I've done everything right, since this is my first PR to adapt_*
One thing that I _suggest_ for translation stuff... is maybe the language codes used should indicate the country too...e.g. en_us, en_uk, es_es, es_ar etc. ... to allow  for different variants of the same language to co-exist. Just a thought.
If I did something wrong or I need to do the PR differently let me know. Thx.
